### PR TITLE
daemonize: new, 1.7.8

### PR DESCRIPTION
**Pull request by neruthes**
Topic Description
-----------------

New packages:

- daemonize

Package(s) Affected
-------------------

- daemonize

Security Update?
----------------

No

Architectural Progress
----------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-utils/daemonize/spec [hunk 1]
```diff
@@ -0,0 +1,4 @@
+VER=1.7.8
+GITSRC="https://github.com/bmc/daemonize.git"
```
Comment (MingcongBai) [inline extra-utils/daemonize/spec]: Please use the new ACBS [spec format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/), here's a rewrite of this file, for your reference.

```
VER=1.7.8
SRCS="git::commit=tags/release-$VER::https://github.com/bmc/daemonize.git"
CHKSUMS="SKIP"
```

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-utils/daemonize/spec (hunk 1 above)
Comment (neruthes) [reply extra-utils/daemonize/spec]: ack

Comment (neruthes) [reply extra-utils/daemonize/spec]: fixed

**Review by MingcongBai — APPROVED** [review]

File: extra-utils/daemonize/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=daemonize
+PKGDES="Command line utility to run a program as a Unix daemon"
+PKGDEP="glibc"
+PKGSEC=utils
+RECONF=0
+
+ABHOST=noarch
```
Comment (MingcongBai) [inline extra-utils/daemonize/autobuild/defines]: My apologies, but it seems contradictory that this package depends on `glibc`, while being `noarch`. What is going on?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-utils/daemonize/autobuild/defines (hunk 2 above)
Comment (neruthes) [reply extra-utils/daemonize/autobuild/defines]: Let me check the details...

**Review by MingcongBai — APPROVED** [review]

**Comment by neruthes** [comment]
Pushed for all 4 architectures.

**Comment by MingcongBai** [comment]
@neruthes Thank you!

**Comment by MingcongBai** [comment]
@neruthes If you could, please submit a test case along with expected results.

**Pull request merged**